.gitignore

```diff
@@ -109,7 +109,6 @@ ENV/
 
 # Binary output and IDEs
 *.pdf
-*.h5
```

Comment:
No need for this here I think. We can exclude hdf5 in general but you can still override per-file.